policy/modules/xen/stubdom.if

```diff
@@ -88,8 +88,7 @@ interface(`stubdom_ioemu',`
 
 	allow stubdom_t $1:resource { add remove };
 	allow stubdom_t $1:domain { getdomaininfo set_target shutdown settime };
-	allow stubdom_t $1:hvm { cacheattr getparam dm
-				 setparam hvmctl };
+	allow stubdom_t $1:hvm { getparam dm setparam hvmctl };
```

Comment:
`cacheattr` goes away because it is now covered by `dm`?

Reply:
Yes